scripts/funcotator/data_sources/getDbSNP.sh

```diff
@@ -154,57 +154,60 @@ function downloadAndVerifyVcfFiles() {
+    md5File=$( cat ${listingFile} | awk '{print $9}' | grep -E "${MD5_FILE_REGEX}" )
 
     echo "${indentSpace}Retrieving MD5 sum file: ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${md5File} ... "
     wget ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${md5File}
 
     # Get the VCF file, then make sure that the contig names are correct for HG19 (if applicable)
     echo "${indentSpace}Retrieving VCF file: ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${vcfFile} ... "
-    if [[ "${filePrefix}" == "hg19" ]] ; then
-        curl ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${vcfFile} | gunzip | sed -e 's#^\([0-9][0-9]*\)#chr\1#' -e 's#^MT#chrM#' -e 's#^X#chrX#' -e 's#^Y#chrY#' | bgzip > ${vcfFile}
-    else
-        wget ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${vcfFile}
-
-        echo "${indentSpace}Retrieving VCF Index file: ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${tbiFile} ... "
-        wget ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${tbiFile}
-
-        # We can only verify the checksum with hg38 because we modify the hg19 file as we stream it in:
-        echo "${indentSpace}Verifying VCF checksum ..."
-        if [[ "$(uname)" == "Darwin" ]] ; then
-            which md5sum-lite &> /dev/null
-            r=$?
-            if [ $r == 0 ] ; then
-                checksum=$( md5sum-lite ${vcfFile} | awk '{print $1}' | sed -e 's#^[ \t]*##g' -e 's#[ \t]*$##g' )
-                expected=$( head -n1 ${md5File} | awk '{print $1}' | sed -e 's#^[ \t]*##g' -e 's#[ \t]*$##g' )
-
-                if [[ "${checksum}" != "${expected}" ]] ; then
-                    error "DOWNLOADED FILE IS CORRUPT!  (${checksum} != ${expected})"
-                    error "FAILING"
-                    exit 4
-                fi
-            else
-                error "Unable to validate md5sum of file: cannot locate 'md5sum-lite'.  Use these data with caution."
-            fi
-        else
-            which md5sum &> /dev/null
-            r=$?
-            if [ $r == 0 ] ; then
-                checksum=$( md5sum ${vcfFile} | awk '{print $1}' | sed -e 's#^[ \t]*##g' -e 's#[ \t]*$##g' )
-                expected=$( head -n1 ${md5File} | awk '{print $1}' | sed -e 's#^[ \t]*##g' -e 's#[ \t]*$##g' )
-
-                if [[ "${checksum}" != "${expected}" ]] ; then
-                    error "DOWNLOADED FILE IS CORRUPT!  (${checksum} != ${expected})"
-                    error "FAILING"
-                    exit 4
-                fi
-            else
-                error "Unable to validate md5sum of file: cannot locate 'md5sum'.  Use these data with caution."
-            fi
-        fi
-    fi
+    curl ftp://ftp.ncbi.nih.gov/snp/organisms/${remoteFolder}/VCF/${vcfFile} | gunzip | sed -e 's#^\([0-9][0-9]*\)#chr\1#' -e 's#^MT#chrM#' -e 's#^X#chrX#' -e 's#^Y#chrY#' | bgzip > ${vcfFile}
```

Comment:
Is `sed` really the best mechanism to do this transformation? Is there a more robust option we could consider (like a liftover file)?

Reply:
A liftover seemed like overkill to me.  The only difference here is the contig names, rather than any actual content, so I think using `sed` is appropriate.